backend-api/src/functions/asyncActiveSession/asyncActiveSessionHandler.ts

```diff
@@ -34,16 +34,28 @@ export async function lambdaHandlerConstructor(
 
   const serviceToken = authorizationHeaderResult.value;
 
+  const decryptResult = await dependencies
+    .jweDecryptor(config.ENCRYPTION_KEY_ARN)
+    .decrypt(serviceToken);
+
+  if (decryptResult.isError) {
+    logger.log("JWE_DECRYPTION_ERROR", {
+      errorMessage: decryptResult.value.errorMessage,
+    });
+    return badRequestResponse("Failed decrypting service token JWE");
+  }
+
+  const jwt = decryptResult.value;
+
   const tokenServiceDependencies = dependencies.tokenServiceDependencies;
   const tokenService = dependencies.tokenService(tokenServiceDependencies);
   const getSubFromTokenResult = await tokenService.getSubFromToken(
     config.STS_JWKS_ENDPOINT,
-    config.ENCRYPTION_KEY_ARN,
-    serviceToken,
     {
       maxAttempts: 3,
       delayInMillis: 100,
     },
+    jwt,
```

Comment:
Passing the JWT to the token service here for now until we've sorted out what's happening to the TokenService

Reply:
Could call this `decryptedServiceToken`?

Reply:
I've changed to `serviceTokenJwe` and `serviceTokenJwt` - is that ok?